doc/guides/west/release-notes.rst

```diff
@@ -1,6 +1,91 @@
 West Release Notes
 ##################
 
+v0.6.x
+******
+
+- No separate bootstrapper
+
+  In west v0.5.x, the program was split into two components, a bootstrapper and
+  a per-installation clone. The bootstrapper was a program that implemented the
+  :ref:`west init <west-multi-repo-cmds>` command, which cloned the west
+  repository itself along with its other tasks. Except for ``west init``, all
+  other west commands were implemented by code in the cloned west repository,
+  which the bootstrapper delegated to. The bootstrapper itself was distributed
+  via PyPI and installed using ``pip3``.
+
+  This is similar to how Google's Repo tool works, but was determined to be a
+  mistake for west.
```

Comment:
"mistake" is a strong word. It served its purpose (quick iteration), so I would rephrase this as "west is now stable enough to be entirely distributed through PyPI"